R/data.R

```diff
@@ -1,11 +1,11 @@
 #' A dataset of sample people based on Fluent UI examples
 #'
 #' @source \url{https://developer.microsoft.com/en-us/fluentui#/controls/web/peoplepicker}
-"fluent_people"
+"fluentPeople"
```

Comment:
interesting, why this change? in general the snake_case is the standard in R, that's why I made it this way. Unfortunately Shiny chose a different convention and therefore for UI components we used camelCase